vnext/Shared/FeatureGate.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+void SetFeatureGate(std::string &&name) noexcept;
```

Comment:
>SetFeatureGate [](start = 5, length = 14)

This should include bool value.

Reply:
This is needed because FeatureGate can be present, yet it's value could be set to false. To avoid callers check for such "false" value, this Api should just take bool value as a parameter.

---
In reply to: [451659738](https://github.com/microsoft/react-native-windows/pull/5458#discussion_r451659738) [](ancestors = 451659738)

Reply:
I was on the fence about that, so initially went with a simplistic approach.

My only concern is:
Are we OK with returning `false` for non-existing FG engries?

Reply:
Yes, "false" is correct default for non-existing FeatureGates.

One thing to be aware of is that in Office we also have ChangeGates. ChangeGates return true by default. Since you are not implementing ChangeGates here, this shouldn't matter for this PR.

---
In reply to: [451922316](https://github.com/microsoft/react-native-windows/pull/5458#discussion_r451922316) [](ancestors = 451922316)